testing/cloud_layouts/script.sh

```diff
@@ -670,9 +719,9 @@ function bootstrap() {
 export PATH="/opt/deckhouse/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 export LANG=C
 set -Eeuo pipefail
-kubectl -n d8-cloud-instance-manager get machines.machine.sapcloud.io
-kubectl -n d8-cloud-instance-manager get machines.machine.sapcloud.io -o json | jq -re '.items | length > 0' >/dev/null
-kubectl -n d8-cloud-instance-manager get machines.machine.sapcloud.io -o json|jq -re '.items | map(.status.currentStatus.phase == "Running") | all' >/dev/null
+kubectl get instance -o json >/dev/null
+kubectl get instance -o json | jq -re '.items | length > 0' >/dev/null
+kubectl get instance -o json | jq -re '.items | map(.status.currentStatus.phase == "Running") | all' >/dev/null
```

Comment:
Is it same for mcm and cluster api both?

Reply:
I think so, I tested in this pr openstack e2e and it ran successfully